### ISSUE TITLE
[Feat] Have the workspace selection dialog box remember the last directory used

### ISSUE THREAD

**Issue by ericrpatton** [OSGeo/grass#2134]
**Is your feature request related to a problem? Please describe.**
**A clear and concise description of what the problem is.**
I would find it useful if the wxGUI workspace selection dialog box would 'remember' the last directory where a file workspace gxw file was opened. As it is right now, I believe it places you in the home directory, but my workspace files are buried several directories down in a Grass database.

**Describe the solution you'd like**
When opening a workspace file, the wxGUI opens in the directory of the gxw file from the previous Grass session (or, at least provides the option to do so in the GUI settings if you don't want to hard-code this behaviour.)

**Describe alternatives you've considered**
The only alternative I see is movning all of my gxw files to my home directory, but that is inconvenient as it would then be outside the rest of the Grass project files.

**Additional context**

**Comment by ericrpatton** [comment]
I never noticed that recent files was available in the workplace selector! We can close this request I think.

**Comment by petrasovaa** [comment]
Sorry for misunderstanding, this is not yet implemented for workspace files, only in the Python editor:
![Untitled](https://user-images.githubusercontent.com/7494312/151486182-5d84907b-4b5a-44c8-ba6c-04de173bb16d.png)

**Comment by ericrpatton** [comment]
It seems I have a 'Recently used' selector in my Grass instance...perhaps a feature of my distro's window manager?

![Workspace](https://user-images.githubusercontent.com/19192513/151583366-847c4bb4-11b8-47e5-bf0a-8be1fb9297d5.png)

**Comment by petrasovaa** [comment]
> It seems I have a 'Recently used' selector in my Grass instance...perhaps a feature of my distro's window manager?

Yes, that's unrelated to GRASS

**Comment by ericrpatton** [comment]
> Yes, that's unrelated to GRASS

Yes, I know, but since the OS window manger is supplying the required functionality, then maybe the feature doesn't need to be implemented separately in Grass, which is my point.
